examples/seismic/tutorials/04_dask.ipynb

```diff
@@ -324,11 +342,17 @@
     "\n",
     "    geometry, rec = load_shot_data(param['shot_id'], dt)\n",
     "    geometry.model = model0\n",
-    "    # Set up solver.\n",
-    "    solver = AcousticWaveSolver(model0, geometry, space_order=4)\n",
+    "    # Set up solver -- load the solver used above in the genration of the syntethic data.    \n",
+    "    with open(\"arguments.pkl\", \"rb\") as cp_file:\n",
+    "        cp = pickle.load(cp_file)\n",
+    "        \n",
+    "    op = cp['operator']\n",
+    "    op.model=model0\n",
+    "    op.geometry=geometry\n",
+    "    op.dt=dt\n",
     "\n",
     "    # Compute simulated data and full forward wavefield u0\n",
-    "    d, u0, _ = solver.forward(save=True)\n",
+    "    d, u0, _ = op.forward(save=True)\n",
```

Comment:
Did we just name a `AcousticWaveSolver` object as `op`? That sounds confusing.

Reply:
I can keep the old name to avoid confusion